Gem/Code/CMakeLists.txt

```diff
@@ -193,6 +193,6 @@ if(PAL_TRAIT_BUILD_UNIFIED_SUPPORTED)
     set_property(GLOBAL APPEND PROPERTY LY_LAUNCHER_UNIFIED_PROJECTS MultiplayerSample)
 endif()
 
-set_property(TARGET MultiplayerSample APPEND PROPERTY GAMELAUNCHER_ADDITIONAL_VS_DEBUGGER_COMMAND_ARGUMENTS "--console-command-file=\"client.cfg\"")
-set_property(TARGET MultiplayerSample APPEND PROPERTY SERVERLAUNCHER_ADDITIONAL_VS_DEBUGGER_COMMAND_ARGUMENTS "--console-command-file=\"server.cfg\"")
-set_property(TARGET MultiplayerSample APPEND PROPERTY UNIFIEDLAUNCHER_ADDITIONAL_VS_DEBUGGER_COMMAND_ARGUMENTS "--console-command-file=\"server.cfg\"")
\ No newline at end of file
+set_property(TARGET MultiplayerSample APPEND PROPERTY GAMELAUNCHER_ADDITIONAL_VS_DEBUGGER_COMMAND_ARGUMENTS "--console-command-file=\"launch_client.cfg\"")
```

Comment:
Would love someone to chime on this. Will be something that could impact play testing.

Reply:
This only affects what occurs when debugging in Visual Studio when running the game launcher. GameLauncher.exe won't be affected.

Reply:
I think the actual question here is if we're ok renaming "server.cfg" to "launch_server.cfg". It affects play testing in the sense that we'll need to change our playtest runbook to account for the new name.